sites/themes.skeleton.dev/src/lib/components/common/Lightswitch/Lightswitch.svelte

```diff
@@ -1,6 +1,9 @@
 <script lang="ts">
```

Comment:
Our goal with Cookbook guides is to show minimal examples to keep people's focus on the most critical information. In this case "how does a lightswitch work", instead of "Skeleton's conventions for prop abstraction".

I'd recommend reverting this change and inlining the attribute values instead. If users wish to abstract this, they can. But this is more boilerplate we add, the harder it can be for learners to parse the critical info.